wrappers/ortools.rb

```diff
@@ -366,7 +376,8 @@ def solve(vrp, job, thread_proc = nil, &block)
             OrtoolsVrp::Capacity.new(
               limit: (q&.limit && q.limit < 1e+22) ? q.limit : -1,
               overload_multiplier: q&.overload_multiplier || 0,
-              counting: unit&.counting || false
+              counting: unit&.counting || false,
+              initial: q.initial || [sum_quantities[unit.id], q.limit].compact.max
```

Comment:
I suspect this `max` should be `min`, right?


Can there be a reason to take the max -- i.e., why would we load the vehicle more than the total_demand if the capacity.limit > total_demand?

Similarly if the the vehicle capacity is less than total_demand, then initial load is set to total_demand > capacity.limit.

Reply:
if we have an overload_multiplier, we can allow the sum of the quantities loaded to exceed the capacity, the `max` is used at this purpose